buzzmoon-backend/back4app-cloud-code/createGame.js

```diff
@@ -0,0 +1,82 @@
+ // this function validates the parameters of a createGame request to ensure
+ // that only valid games are created
+ const validateCreateGameParams = (request) => {
```

Comment:
In addition to checking if the audio is in fact audio (and not PDF, any file), are you error handling if the file is too big? consider both throwing and letting the user know up-front what the max file size is